src/patternfly/base/_shield-inheritable.scss

```diff
@@ -25,7 +25,7 @@
   .pf-c-drawer,
   .pf-c-dropdown,
   .pf-c-empty-state,
-  .pf-c-expandable,
+  .pf-c-expandable-section,
```

Comment:
Not sure if a merge went awry, but why is this showing up in the diff? :(

Reply:
mmmm, good question. let me double heck